tests/sympc/tensor/share_tensor_test.py

```diff
@@ -20,6 +22,22 @@ def test_send_get(get_clients, precision, base) -> None:
     assert x_share == x_ptr.get()
 
 
+@pytest.mark.parametrize("precision", [12, 3])
+@pytest.mark.parametrize("base", [4, 6])
+def test_send_get_orchestrator(get_clients, precision, base) -> None:
```

Comment:
Are precision and base parameters required? If not remove the pytest parameterize arguments too.